frontend/packages/console-app/src/components/cloud-shell/CloudshellTerminal.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import { referenceForModel, k8sCreate, k8sList } from '@console/internal/module/k8s';
+import { LoadingBox, Firehose, FirehoseResource } from '@console/internal/components/utils';
+
+import { WorkspaceModel } from '../../models';
+import CloudshellResource from './CloudshellResource';
+import { newCloudShellWorkSpace, CloudShellResource } from './utils/cloudshell-resource';
+
+type CloudShellTerminalProps = {};
+
+const CloudShellTerminal: React.FC<CloudShellTerminalProps> = () => {
+  /* Change dedicated name and namespace as per update */
+  const namespace = 'che-workspace-controller';
+  const name = 'cloudshell-userID';
+  const [cloudShell, setCloudShell] = React.useState<CloudShellResource>(null);
+
+  const createCloudShell = () => {
+    k8sCreate(WorkspaceModel, newCloudShellWorkSpace(name, namespace))
+      .then((newShell) => {
+        setCloudShell(newShell);
+      })
+      .catch((error) => {
+        // eslint-disable-next-line no-console
+        console.warn('Cloudshell creation Error', error);
+      });
```

Comment:
IMO, If there is an error we should show it to the user else cloud shell will be stuck in the loading state.
 we can show it in either cloud shell body or in a modal. Please, verify with UX.

cc @christianvogt